src/sqlfluff/rules/L053.py

```diff
@@ -71,7 +71,7 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             *[
                 segment
                 for segment in context.segment.segments
-                if segment.name not in bracket_set and not segment.is_meta
+                if segment.get_type() not in bracket_set and not segment.is_meta
```

Comment:
Have we (or could we) document somewhere the difference between `.is_type(a, b, c)` versus `.get_type() in {a, b, c}`. I assume it's pretty subtle but also pretty important. (And I'm not sure I know the difference myself.)

Reply:
- `.get_type() in {a, b, c}` only checks the last type in the stack.
- `.is_type(a, b, c)` checks against all parent types.